srcpkgs/pulseaudio/template

```diff
@@ -10,15 +10,16 @@ configure_args="--disable-oss-output --disable-oss-wrapper --disable-tcpwrap
  --disable-systemd-login --disable-systemd-daemon --disable-systemd-journal
  --enable-webrtc-aec --with-bash-completion-dir=/usr/share/bash-completion/completions
  --enable-bluez5-native-headset"
-hostmakedepends="automake gettext-devel intltool libtool orc-devel pkg-config"
+hostmakedepends="automake autoconf gettext-devel glib-devel intltool libtool
+orc-devel pkg-config"
```

Comment:
missing indention here

Reply:
Guess I better add it to the other dependency blocks as well then